client/src/components/Misc/EmptyState.tsx

```diff
@@ -0,0 +1,22 @@
+import { Spinner, Intent } from '@blueprintjs/core';
+import React, { memo } from 'react';
+
+import css from './EmptyState.scss';
+
+export interface Props {
+  text: string;
+  spinnerSize?: number;
+  spinnerIntent?: Intent;
+}
+
+export const EmptyState = memo<Props>(function EmptyState(props) {
```

Comment:
`EmptyState` doesnt say anything related to what it does
`LoadingOverlay` could be better